.travis.yml

```diff
@@ -147,9 +147,21 @@ jobs:
     - travis_retry scripts/deploy-to-gcs.sh
     - travis_retry scripts/trigger-build-helm-charts.sh
 
+  - name: Deploy to Demo Environment
+    #if: type in (push, api) AND tag is present
+    install:
+    - travis_retry scripts/setup-tools-for-ubuntu.sh
+    script:
+    # travisqueue.go ensures only a single build runs at a time on Openshift
+    - go run travisqueue.go
+    - travis_retry scripts/setup-openshift.sh ${OC_DEMO_TOKEN}
+    - TILLER_NAMESPACE=console-demo enterprise-suite/scripts/lbc.py install --namespace=console-demo --local-chart=enterprise-suite --set usePersistentVolumes=true,defaultStorageClass=gp2
+
 branches:
   only:
   - master
+  # build tags starting with `v`, e.g. v0.0.16
+  - /^v.*$/
```

Comment:
Doubt it really matters, but that would accept "vagabond" etc. too...

Reply:
Yeah I don't think we need to worry about that.

However I just realize we don't use semver for our console charts tag (I think because we originally thought it would have multiple charts). As a result this won't work. So need to think about that.

Reply:
How is our tag not semver compliant?  I see tags like:
```
enterprise-suite-1.0.0
enterprise-suite-1.0.0-rc.2
enterprise-suite-1.0.0-rc.3
enterprise-suite-1.0.0-rc.4
enterprise-suite-1.0.0-rc.5
enterprise-suite-1.0.0-rc.6
enterprise-suite-1.0.0-rc.7
enterprise-suite-1.0.1
```
and those all look good to me.  Is there a new tag now?